applications/crossbar/doc/ledgers.md

```diff
@@ -108,6 +108,84 @@ curl -v -X GET \
     http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/ledgers/{SOURCE_SERVICE}
 ```
 
+## Get ledgers summary grouped by sub-accounts
+
+> GET /v2/accounts/{ACCOUNT_ID}/ledgers/summary_by_accounts
+
+```shell
+curl -v -X GET \
+    -H "X-Auth-Token: {AUTH_TOKEN}" \
+    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/ledgers/summary_by_accounts
+```
+
+**Sample response:**
+
+```json
+{
+    "auth_token": "{AUTH_TOKEN}",
+    "data": {
+        "{ACCOUNT_ID_1}": {
+            "per-minute-voip": {
```

Comment:
@icehess I think we should put the ledgers under an object (such as 'ledgers') to seperate the list of ledger sources from the 'account' container/object.

Reply:
done. Also I add total just in case :)